components/dashboard/src/admin/Settings.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useContext } from "react";
+import { InstallationAdminSettings } from "@gitpod/gitpod-protocol";
+import { AdminContext } from "../admin-context";
+import CheckBox from "../components/CheckBox";
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { getGitpodService } from "../service/service";
+import { adminMenu } from "./admin-menu";
+
+export default function Settings() {
+    const { adminSettings, setAdminSettings } = useContext(AdminContext);
+
+    const actuallySetTelemetryPrefs = async (value: InstallationAdminSettings) => {
+        await getGitpodService().server.adminUpdateSettings(value);
+        setAdminSettings(value);
+    }
+
+    return (
+        <div>
+            <PageWithSubMenu subMenu={adminMenu} title="Settings" subtitle="Configure settings for your Gitpod cluster.">
+                <CheckBox
+                    title="Enable Service Ping"
+                    desc={<span>This is used to provide insights on how you use your cluster so we can provide a better overall experience. Read our <a className="gp-link" href="https://www.gitpod.io/privacy">Privacy Policy</a></span>}
```

Comment:
**suggestion(non-blocking)**: To re-use the pattern of the rest of help links (_Learn more_ links) across the product we could hyperlink the whole action here. What do you think? 

```suggestion
                    desc={<span>This is used to provide insights on how you use your cluster so we can provide a better overall experience. <a className="gp-link" href="https://www.gitpod.io/privacy">Read our privacy policy</a></span>}
```

Reply:
You're the boss when it comes to the front-end - I'm happy to oblige with this